examples/tokens/custom_royalty_fee.py

```
@@ -0,0 +1,129 @@
+    except (TypeError, ValueError) as e:
+        print(f"Error: {e}")
+        print("Please check OPERATOR_ID and OPERATOR_KEY in your .env file.")
+        sys.exit(1)
+
+def custom_royalty_fee_example():
+    """Demonstrates how to create a token with a custom royalty fee."""    
+    
+    client, operator_id, operator_key = setup_client()
+    print(f"Connecting to the Hedera {network_name} network")
+    
+    with Client(Network(network_name)) as client:
```

Ruff (0.14.8)

Redefinition of unused `client` from line 55

(F811)